apps/stops/fixture/vcr_cassettes/elixir_stops_apitest__parses_parent_id_and_child_ids.json

```diff
@@ -0,0 +1,242 @@
+      "headers": {
+        "Date": "Tue, 05 May 2020 18:44:45 GMT",
+        "Content-Type": "application/vnd.api+json; charset=utf-8",
+        "Content-Length": "28475",
+        "Connection": "keep-alive",
+        "cache-control": "max-age=0, private, must-revalidate",
+        "content-encoding": "gzip",
+        "last-modified": "Tue, 05 May 2020 13:55:01 GMT",
+        "server": "Cowboy",
+        "strict-transport-security": "max-age=31536000",
+        "vary": "accept-encoding",
+        "x-ratelimit-limit": "6944",
+        "x-ratelimit-remaining": "6897",
+        "x-ratelimit-reset": "1588704300",
+        "x-request-id": "Fgw2IWymV6hIUwAABgsh"
+      },
+      "status_code": 200,
+      "type": "ok"
+    }
+  },
+  {
+    "request": {
+      "body": "",
+      "headers": {
+        "accept-encoding": "gzip",
+        "accept": "application/vnd.api+json",
+        "x-api-key": "1c05fc0118144698888cc5a1b08d3a56"
```

Comment:
This shouldn't be checked in. I'm going to disable this key and I'll share a different one with you.

Are there other API keys in the other cassettes?

Reply:
@paulswartz it's supposed to filter that header out, and in fact in other places in this same cassette it does so--this is very odd. Possibly a bug in ExVCR?